ckanext/datastore/backend/postgres.py

```diff
@@ -1166,8 +1165,10 @@ def upsert_data(context: Context, data_dict: dict[str, Any]):
             raise ValidationError({
                 'table': [u'table does not have a unique key defined']
             })
-
         for num, record in enumerate(records):
+            # remove the _id if not provided when calling update method
+            if '_id' in unique_keys and '_id' not in record:
+                unique_keys.remove('_id')
```

Comment:
If `_id` is the only key in `unique_keys` then this would allow "upserts" without an `_id` specified in `record`, right?